app/components/app_patient_medical_history_card_component.html.erb

```diff
@@ -0,0 +1,26 @@
+<% if has_medical_history? %>
```

Comment:
Should the component be the judge of whether it should show or not? I'm thinking this could live at the call-site.

Reply:
Ya, I think I was thinking to accommodate displaying an alternative if the health questions aren't there, but that would be better placed with the caller. Will change.